presto-main/src/main/java/com/facebook/presto/sql/planner/QueryPlanner.java

```diff
@@ -351,6 +355,87 @@ private PlanBuilder project(PlanBuilder subPlan, Iterable<Expression> expression
                 analysis.getParameters());
     }
 
+    private PlanBuilder project(
+            PlanBuilder subPlan,
+            ImmutableList<FilterExpressionWithAggregate> filterExpressionWithAggregates,
+            ImmutableList<Expression> groupAndSortArguments,
+            ImmutableList.Builder argumentsAfterFilter)
```

Comment:
* Use `List` here, not `ImmutableList` nor `ImmutableList.Builder`.
* `ImmutableList.Builder argumentsAfterFilter` you are missing type here